### PR TITLE
[PPP-4620] - Use of Vulnerable Component: Components/xmlrpc-client:3.1.3

### PR DESCRIPTION
**Pull request by andreramos89**
Removal of OpenERP connector from PRD - validated by CSS team no client is using this feature

----------------

@smmribeiro 
@renato-s 
@bcostahitachivantara

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 45m 6s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **2646**, Failures: **0**, Skipped: **15** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/reporting/job/PR-1580/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :x: Build failed in 45m 46s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **2545**, Failures: **0**, Skipped: **14** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/reporting/job/PR-1580/2/testReport)

 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

File: pom.xml [hunk 1]
```diff
@@ -170,6 +169,7 @@
     <pentaho-metaverse.version>10.1.0.0-SNAPSHOT</pentaho-metaverse.version>
     <encryption-support.version>10.1.0.0-SNAPSHOT</encryption-support.version>
     <jakarta-annotation-api.version>1.3.5</jakarta-annotation-api.version>
+    <ws-commons-util-version>1.0.2</ws-commons-util-version>
```
Comment (bcostahitachivantara) [inline pom.xml:172]: Why have you added this property?

Comment (andreramos89) [reply pom.xml]: Compilation error in the day of local build. Already solved. No longer needed

**Comment by buildguy** [comment]
### :x: Build failed in 44m 2s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **2545**, Failures: **0**, Skipped: **14** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/reporting/job/PR-1580/3/testReport)

 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Review by smmribeiro — CHANGES_REQUESTED** [review]
Wingman is complaining over the code under "/designer/datasource-editor-openerp" as that uses code that is deleted on this PR (under "/engine/extensions-openerp").
Shouldn't that "tree" also be deleted?

**Comment by buildguy** [comment]
### :x: Build failed in 46m 57s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **2545**, Failures: **0**, Skipped: **14** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/reporting/job/PR-1580/4/testReport)

 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by andreramos89** [comment]
> Wingman is complaining over the code under "/designer/datasource-editor-openerp" as that uses code that is deleted on this PR (under "/engine/extensions-openerp"). Shouldn't that "tree" also be deleted?

In fact that was missing. Thanks

**Review by smmribeiro — APPROVED** [review]
Looks good.

**Pull request merged**